src/server/Group.cpp

```diff
@@ -25,6 +25,17 @@ void Group::update() {
             });
         mCircle->setVelocity(new_velocity * GROUP_SPEED);
 
+        // group is groupable if any member player wants to group
+        // should probably switch to voting functionality later
+        bool groupable = std::accumulate(
+            mMembers.begin(),
+            mMembers.end(),
+            false,
+            [](bool curr, std::shared_ptr<Player> player) {
+                return curr || + player->getGroupable();
```

Comment:
woah typo the + shouldn't be there. wtf this works o_0

Reply:
fix plox